java/src/org/openqa/selenium/edge/EdgeDriverService.java

```diff
@@ -148,6 +154,14 @@ public EdgeDriverService.Builder withVerbose(boolean verbose) {
       return this;
     }
 
+    /**
+     * Configures the driver server log level.
+     */
+    public EdgeDriverService.Builder withLoglevel(String level) {
```

Comment:
FYI: When users provide an invalid log level, MsEdgeDriver will be terminated due to an error; `Invalid --log-level value.`

```
$ ./msedgedriver --log-level=UNKNOWN
Starting Microsoft Edge WebDriver 104.0.1293.63 (7dd22b4a7f122e6953a166b0388edf494d716c7d) on port 9515
To submit feedback, report a bug, or suggest new features, please visit https://github.com/MicrosoftEdge/EdgeWebDriver

Only local connections are allowed.
Please see https://aka.ms/WebDriverSecurity for suggestions on keeping Microsoft Edge WebDriver safe.

Invalid --log-level value.
Unable to initialize logging. Exiting...
```

I wondered if I should validate if a given value is valid, but I didn't implement it because I was concerned that support costs would be high. For example, it is an extreme case, but we might need to have a log level list for each version when MsEdgeDriver adds or removes log levels, etc.